sdk/python/foundation-models/huggingface/inference/stable_diffusion/text-to-image-online-endpoint.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
Instead of sdk/foundation-models/system/inference/stable_diffusion; let's use cli/foundation-models/system/inference/text-to-image

This is to align with the naming convention that is used for other tasks